### ISSUE TITLE
Full-page errors

### ISSUE THREAD

**Issue by elycheea** [carbon-design-system/ibm-products#4009]
# Full page errors

## Brief description

Display a full-page error when the requested page is unavailable to the user. This is typically caused by issues with the requested URL or access permissions. Errors caused by server connectivity issues are not covered in this guideline.

## Details

This pattern is intended to replace the current version of HTTP errors.

[Link to design](https://pages.github.ibm.com/cdai-design/pal/patterns/full-page-errors/usage/)
[Current version](https://ibm-products.carbondesignsystem.com/?path=/docs/ibm-products-patterns-http-errors-httperror403--docs) (3 variants — 403, 404, other)

### Links to other materials

e.g. sketch files, icons, images, prototypes

## Design Maintainer(s)

- @thefirstartist 

## Tasks

Before starting work on this epic, please review and complete the following.

- [x] Initial review of design/existing code

### Working in Carbon for IBM Products package

- [x] Have you reviewed our
      [contribution guidelines](https://github.com/carbon-design-system/ibm-products/blob/master/.github/CONTRIBUTING.md)?
- [x] Have you noted our
      [code guidelines](https://github.com/carbon-design-system/ibm-products/blob/master/docs/CODE_GUIDELINES.md)?
- [x] Did you use the CLI to scaffold your new component?

```[tasklist]
### Acceptance criteria
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4075
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4080
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4138
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4081
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4402
- [ ] https://github.com/carbon-design-system/carbon-for-products-design-kit/issues/23
- [ ] https://github.com/carbon-design-system/ibm-products/issues/5050
```

```[tasklist]
### Review process
- [ ] #4434
- [ ] #4435
- [ ] #4439
- [ ] https://github.com/carbon-design-system/ibm-products/issues/4083
```

**Comment by elycheea** [comment]
Given @davidmenendez’s recent refactor of empty state, I would suggest that we set a call with at least @thefirstartist @davidmenendez to define the acceptance criteria for this issue.

**Comment by elycheea** [comment]
@devadula-nandan Not that I was expecting to see many issues with full-page errors, but happy to share that accessibility review passed with only one non-blocking issue! Assigned to you but don’t think you need to address it right away either since it’s a lower priority cleanup. We have the tiniest bit of redundant code in Storybook. 🙂
